pkg/runtime/wfengine/actor_backend.go

```diff
@@ -110,24 +117,41 @@ func getActivityScheduler(activityWorkItemChan chan *backend.ActivityWorkItem) a
 }
 
 // InternalActors returns a map of internal actors that are used to implement workflows
-func (be *actorBackend) GetInternalActorsMap() map[string]actors.InternalActor {
+func (abe *ActorBackend) GetInternalActorsMap() map[string]actors.InternalActor {
 	internalActors := make(map[string]actors.InternalActor)
-	internalActors[be.config.workflowActorType] = be.workflowActor
-	internalActors[be.config.activityActorType] = be.activityActor
+	internalActors[abe.config.workflowActorType] = abe.workflowActor
+	internalActors[abe.config.activityActorType] = abe.activityActor
 	return internalActors
 }
 
-func (be *actorBackend) SetActorRuntime(actors actors.Actors) {
-	be.actors = actors
+func (abe *ActorBackend) SetActorRuntime(actorRuntime actors.ActorRuntime, ctx context.Context) {
```

Comment:
nit: I believe it's a common Go convention that the `ctx` parameter be the first parameter in a function signature.